utils/preferences/tests/dtf/data_provider.rs

```diff
@@ -0,0 +1,36 @@
+use super::*;
+use icu_datetime::options::preferences::HourCycle;
+use icu_locid::{langid, LanguageIdentifier};
+use tinystr::tinystr;
+
+struct DefaultPrefs {
+    pub und: DateTimeFormatDefaultPreferences,
+    pub list: &'static [(LanguageIdentifier, DateTimeFormatDefaultPreferences)],
+}
+
+const DEFAULT_PREFS: DefaultPrefs = DefaultPrefs {
+    und: DateTimeFormatDefaultPreferences {
+        hour_cycle: HourCycle::H12,
```

Comment:
Why is `und` `H12` instead of `H23`?

Reply:
Ah, good catch. It's just a test, but fixed.